### DISCUSSION TITLE
gatsby sass

### DISCUSSION THREAD

**Discussion by mf-Fadavi** [gatsbyjs/gatsby#35137] [Help]
I have installed the Gatsby-plugin-sass and also imported the plugin in the gatsby-config.js.
But I get this error.
<img width="1154" alt="Screenshot 2022-03-15 130423" src="https://user-images.githubusercontent.com/100221384/158374165-ead2af15-97a0-438a-a0ef-bfffa37704cb.png">
Should I install an additional plugin for sass on VS-Code, or how can I solve this problem.

**Comment by AnilSeervi** [comment]
Have you installed `sass` along with `gatsby-plugin-sass` ?
